packages/ubuntu_desktop_installer/linux/my_application.cc

```diff
@@ -34,11 +59,16 @@ static void my_application_activate(GApplication* application) {
   gtk_header_bar_set_decoration_layout(header_bar, ":close");
   gtk_window_set_titlebar(window, GTK_WIDGET(header_bar));
   gtk_window_set_default_size(window, 960, 680);
+  gtk_widget_realize(GTK_WIDGET(window));
+
+  if (my_application_fit_to_workarea(window)) {
```

Comment:
If the resolution is changed later, then this may no longer apply. Not sure if this is applicable in the installer use case. I know when I use VMs the resolution normally defaults to too small, and then I immediately resize it.

Reply:
Thanks, this is a great idea. I left #518 open and added a link to this comment.